app/src/main/res/layout/offer_header.xml

```diff
@@ -21,11 +22,14 @@
 
             <TextView
                 android:id="@+id/campaign"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/base_margin_half"
                 android:background="@drawable/background_premium_box_with_campaign"
-                android:textAppearance="?textAppearanceSubtitle1"
+                android:gravity="center"
+                android:paddingBottom="@dimen/base_margin_half"
+                android:textAppearance="?textAppearanceSubtitle2"
+                android:textSize="12sp"
```

Comment:
The text size should be inherited from the text appearance, right? 🤔

Reply:
You are right, added a new style